.github/workflows/bump-push-to-main.yml

```diff
@@ -0,0 +1,39 @@
+name: Bump version on push to main
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    needs: [unit-test-iOS-16, integration-test-iOS-16]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get Previous Tag
+        id: previoustag
+        uses: "WyriHaximus/github-action-get-previous-tag@v1"
+        with:
+          fallback: 0.0.0
```

Comment:
@blaynebayer 
I'm wondering if this should just fail if this can't find the last tag so we don't get into any weird situations?

Reply:
It should never fallback, since we already have existing tags. So I think it's safe. It's also required by that action.